src/config_parse.go

```diff
@@ -178,7 +178,7 @@ func parseCollection(c *collectionParser) ([]*collection, error) {
 				}
 				newIndex := &index{
 					name: indexParser.Name,
-					oid:  indexParser.Oid,
+					oid:  indexOid,
```

Comment:
This is the error I talk about before, since there are not unit tests, this was not added for this case. I found it thanks to the linter @brushknight